docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -33,7 +33,7 @@ Comparing ``loop`` and ``with_*``
 
 you would need::
 
-  loop: [1, [2,3] ,4] | flatten(1)
+  loop: "{{ [1, [2,3] ,4] | flatten(1) }}"
```

Comment:
Running the old code with Ansible resulted in an error. It has to be written in this form to work.

For the example itself, it might be better to rewrite this as
```.yaml
loop: "{{ data | flatten(1) }}"
vars:
  data: [1, [2, 3] ,4]
```
At least that's what I would use in a playbook, because it's much better readable and extensible.